test/param/paramForLoopError.good

```diff
@@ -0,0 +1 @@
+test/param/paramForLoopError.chpl:1: error: for positive count, first index must be present
```

Comment:
I think that's the right approach.

But I think that test won't pass via `start_test` as is.  I think start_test will cd to the directory containing the test, and execute `chpl` from there.  So the error will be reported at `paramForLoopError.chpl:1`, not `test/param/paramForLoopError.chpl:1`, and that .good file won't match.

You can see (lots of) other examples by `find test -name \*.good xargs grep error:`.

Reply:
Okk, I will update the code.